integration-tests/soak/soak_runner_test.go

```diff
@@ -0,0 +1,137 @@
+package soak_test
```

Comment:
Let's please start writing documentation about our integration-tests under `./docs`. Please start with some getting started overview (what is it, how to run it).

Reply:
Adjusted the existing documentation in `docs/integration-tests/README.md`